srcpkgs/starship/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'starship'
 pkgname=starship
-version=0.50.0
-revision=2
+version=0.51.0
+revision=1
 build_style=cargo
 build_helper=qemu
-hostmakedepends="pkg-config libgit2-devel"
-makedepends="openssl-devel libgit2-devel"
```

Comment:
Why remove `libgit2` now?

Reply:
It is probably not picking up because we have 1.0 (see #28456), which means it builds a statically linked libgit2. But `libgit2-devel` also pulls in `zlib-devel`, which is necessary here, otherwise it will also build a statically linked copy of zlib.

Reply:
Ah yes my mistake. I forgot that PR hasn't been merged